src/main/scala/org/opennetworkinsight/dns/DNSPostLDA.scala

```diff
@@ -48,7 +42,7 @@ object DNSPostLDA {
 
     logger.info("Computing conditional probability")
```

Comment:
I'm gonna add a Jira ticket to "fix the logging functinoality in the ML pipeline"